examples/basic/src/actions/index.js

```diff
@@ -3,10 +3,11 @@ import * as Rx from 'rxjs';
 export const FETCH_USER_PENDING = 'FETCH_USER_PENDING';
 export const FETCH_USER_FULFILLED = 'FETCH_USER_FULFILLED';
 export const FETCH_USER_ABORTED = 'FETCH_USER_ABORTED';
+const BASE_URL = 'http://localhost:3000/items/';
 
 export const fetchUser = () => (
-  (actions, store) => Rx.Observable.of({ id: 1, name: 'Bilbo Baggins', timestamp: new Date() })
-    // Delaying to emulate an async request, like Rx.Observable.ajax('/api/path')
+  (actions, store) => Rx.Observable.ajax(BASE_URL)
+    // Delaying to provide enough time to cancel manually
     .delay(1000)
```

Comment:
If we're already delaying ajax requests on the json-server (w/ `--delay 1000`, I think it's best to remove this so it's more "real world".